H5P/laravel-h5p/src/LaravelH5p/Repositories/EditorAjaxRepository.php

```diff
@@ -24,7 +24,7 @@ public function getAuthorsRecentlyUsedLibraries()
         global $wpdb;
         $recently_used = array();
         $user = Auth::user();
-        $user_id = $user->id;
+        $user_id = is_object($user) ? $user->id : 1;
```

Comment:
What is this?

Reply:
yes that was a test code. thanks for the catch.